### ISSUE TITLE
Add replaceText to Title

### ISSUE THREAD

**Issue by MiniDay** [KyoriPowered/adventure#668]
as the title, it is suggested to add `replaceText` to the title
```
  @Contract(pure = true)
  @NotNull Component replaceText(final @NotNull TextReplacementConfig config);
```
allow it to replace text in both title and subtitle

**Comment by kezz** [comment]
I don't like this. I'd be much more in favour of some sort of function that can mutate a title or something. Seems much more useful than this relatively useless utility method.

**Comment by zml2008** [comment]
Yeah, I think this is too specialized as well. A general-purpose title.editAll(), or title.edit(titleConsumer, subtitleConsumer) might make more sense?

**Comment by thiccaxe** [comment]
Perhaps a builder for titles could solve this and serve as a general purpose builder

**Comment by zml2008** [comment]
seems sensible yeah, a title builder would be a good addition
